tests/integration/features/serverless_feature_test.go

```diff
@@ -77,8 +77,9 @@ var _ = Describe("Serverless feature", func() {
 		namespace := envtestutil.AppendRandomNameTo(testFeatureName)
 
 		dsciSpec := newDSCInitializationSpec(namespace)
+		origin := newOrigin("kserve")
 		testFeature, err = feature.CreateFeature(testFeatureName).
-			For(dsciSpec).
+			For(dsciSpec, origin).
```

Comment:
Is that right? Origin is of a type DSCI here?

Reply:
linter got me there -__- fixed, here [103593f](https://github.com/maistra/opendatahub-operator/pull/17/commits/103593f01d204ded9ae413064bddcd5b8e9a5131)